sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/TestUtils.java

```diff
@@ -158,12 +165,21 @@ public static String getConnectionString(boolean withSas) {
     /**
      * Gets the fully qualified domain name for the service bus resource.
      *
-     * @return The fully qualified domain name for the service bus resource.
+     * @return The fully qualified domain name for the service bus resource if set else a redacted dummy name.
      */
     public static String getFullyQualifiedDomainName() {
         return getPropertyValue("AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME", "REDACTED.servicebus.windows.net");
     }
 
+    /**
+     * Gets the fully qualified domain name for the service bus resource.
+     *
+     * @return the fully qualified domain name for the service bus resource if set, {@code null} otherwise.
+     */
+    public static String getLiveFullyQualifiedDomainName() {
```

Comment:
mm I feel like this is better as an overload rather than a newly named method.  I was expecting entirely different results, but it looks like this one doesn't use a default.

`getFullyQualifiedDomainName(boolean useFallbackValue)` or something.

Reply:
nice, this is better, updated.